src/core/server/http/http_server.ts

```diff
@@ -126,14 +155,16 @@ export class HttpServer {
     }
     this.log.debug('starting http server');
 
-    this.setupBasePathRewrite(this.server, config);
-
     for (const router of this.registeredRouters) {
       for (const route of router.getRoutes()) {
+        const isAuthRequired = Boolean(this.authRegistered && route.authRequired);
```

Comment:
are fine to introduce `authRequired` flag as unprotected route definition?

Reply:
I think that's reasonable, as long as the default value is `true`